puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foremanRelease.groovy

```diff
@@ -0,0 +1,108 @@
+pipeline {
+    agent none
+
+    triggers {
+        cron('H 21 * * *')
+    }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+	stages {
+		stage('Mash Koji Repositories') {
+	        agent { label 'sshkey' }
+
+			steps {
+
+				sh "ssh -o 'BatchMode yes' root@koji.katello.org foreman-mash-split.py"
+
+			}
+		}
+        stage('Repoclosure') {
+            steps {
+
+                repoclosure('RHEL', '7')
+
+            }
+        }
+        stage('Install Test') {
+            agent { label 'el' }
+
+            environment {
+                VAGRANT_DEFAULT_PROVIDER = 'rackspace'
+            }
+            steps {
+
+                git url: 'https://github.com/ehelms/forklift', branch: 'foreman-bats'
```

Comment:
Can updated once https://github.com/theforeman/forklift/pull/522 is merged.

Reply:
Another thing to note is that this one doesn't add the koji repos but just the nightlies. Is that intended or an oversight?